patroni/postgresql/slots.py

```diff
@@ -497,12 +502,13 @@ def sync_replication_slots(self, cluster: Cluster, nofailover: bool,
         :returns: list of logical replication slots names that should be copied from the primary.
         """
         ret = []
-        if self._postgresql.major_version >= 90400 and cluster.config:
+        if self._postgresql.major_version >= 90400 and self._postgresql.global_config and cluster.config:
             try:
                 self.load_replication_slots()
 
-                slots = cluster.get_replication_slots(self._postgresql.name, self._postgresql.role,
-                                                      nofailover, self._postgresql.major_version, True)
+                slots = cluster.get_replication_slots(
+                    self._postgresql.name, self._postgresql.role, nofailover, self._postgresql.major_version,
+                    is_standby_cluster=self._postgresql.global_config.is_standby_cluster, show_error=True)
```

Comment:
```suggestion
                    self._postgresql.name,
                    self._postgresql.role,
                    nofailover,
                    self._postgresql.major_version,
                    is_standby_cluster=self._postgresql.global_config.is_standby_cluster,
                    show_error=True,
                )
```

Reply:
The goal is not to keep the code shorter using less space but for me to easily discern between args. It's easier on my eye to scan down a list than to read horizontally with mixed breaks in dividing spaces.

btw I thought this looked a little messy and ran black on it and this was the result.